### ISSUE TITLE
Settings are not synced

### ISSUE THREAD

**Issue by niftyhack** [microsoft/vscode#171537]
Type: <b>Bug</b>

- Freshly install VS Code on MacOS Ventura Laptop
- Turn on Settings Sync
- Turn on Settings Sync on existing VS Code on Debian Laptop

Expectation: Same Extensions on MacOs Ventura and Debian Laptop

Actual: Different Extensions on MacOS Ventura and Debian Laptop

VS Code version: Code 1.74.3 (97dec172d3256f8ca4bfb2143f3f76b503ca0534, 2023-01-09T16:57:40.428Z)
OS version: Linux x64 5.18.0-2-amd64 snap
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-2410M CPU @ 2.30GHz (4 x 2788)|
|GPU Status|2d_canvas: unavailable_software<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: disabled_software<br>multiple_raster_threads: enabled_on<br>opengl: disabled_off<br>rasterization: disabled_software<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: unavailable_software<br>webgl2: unavailable_software<br>webgpu: disabled_off|
|Load (avg)|1, 1, 1|
|Memory (System)|7.55GB (2.99GB free)|
|Process Argv|--no-sandbox|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|lightdm-xsession|
|XDG_CURRENT_DESKTOP|X-Cinnamon|
|XDG_SESSION_DESKTOP|lightdm-xsession|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (28)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-intelephense-client|bme|1.9.4
path-intellisense|chr|2.8.4
gitignore|cod|0.9.0
githistory|don|0.6.19
xml|Dot|2.5.1
gitlens|eam|13.2.0
EditorConfig|Edi|0.16.4
go|gol|0.37.0
autoconf|mae|0.1.0
VS-code-vagrantfile|mar|0.0.7
dotenv|mik|1.0.1
vscode-docker|ms-|1.23.3
vscode-language-pack-de|MS-|1.74.12140926
vscode-kubernetes-tools|ms-|1.3.11
python|ms-|2022.20.2
vscode-pylance|ms-|2023.1.20
jupyter|ms-|2022.11.1003412109
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.13
vscode-jupyter-cell-tags|ms-|0.1.6
vscode-jupyter-slideshow|ms-|0.1.5
cpptools|ms-|1.13.9
material-icon-theme|PKi|4.23.1
vscode-data-preview|Ran|2.3.0
vscode-yaml|red|1.11.0
vscode-scheme|sjh|0.4.0
zig|tie|0.2.6
markdown-all-in-one|yzh|3.5.0

</details>

**Comment by sandy081** [comment]
> Different Extensions on MacOS Ventura and Debian Laptop

Can you please check if you are using same account for syncing? 
What is the difference in extensions you see?

**Comment by niftyhack** [comment]
I uninstalled VS Code from the Debian Laptop, removed the directory `.config/Code/` and installed VS Code again. Problem went away.

> What is the difference in extensions you see?

Some Jupyter extensions that I had removed from the Mac were still present on the Debian Laptop even after multiple syncs.

Initial sync on a third machine worked on the first try today.

I'm closing this, thanks for reaching out!
